docs/src/rules/no-irregular-whitespace.md

```diff
@@ -190,6 +191,21 @@ function thing() {
 }
 ```
 
+### skipJSXText
```

Comment:
The new text is inserted before `:::` so the example above will be missing its closing `:::`.

Reply:
✅